### DISCUSSION TITLE
Unable to connect to Bitrefill, Dial proxy failed

### DISCUSSION THREAD

**Discussion by tcibir** [Ride-The-Lightning/RTL#931] [Troubleshooting Questions] [answered]
Setup: 

Umbrel on Raspberry PI 4 with RTL App
Bought a channel at Bitefill
Adding a peer with Bitrefill fails

**To Reproduce**

Add a peer with 030c3f19d742ca294a55c00376b3b355c3c90d61c6b6b39554dbc7ac19b141c14f@52.50.244.44:9735

Error Message appears when trying to connect with "Add Peer":

"Dial proxy failed: socks connect tcp 10.21.21.11:9050->52.50.244.44:9735: unknown error general socks server failure"

**Screenshots**

![image](https://user-images.githubusercontent.com/85399430/149555746-b3b81cec-385a-4068-9933-2e84e52d5556.png)

**Your environment**

RTL: 0.11.2
Umbrel 0.4.10

**Comment by saubyk** [comment, marked as answer]
Hi @tcibir this has nothing to do with RTL. You'll get the same response, if you try to connect to this peer via command line. I would recommend getting in touch with Bitrefill support and tell them that you're not able to connect your node with theirs.
